stage2/server.py

```diff
@@ -31,11 +31,68 @@ def __repr__(self):
         )
 
 
+class ChatRoom:
+    def __init__(self, room_name):
+        self.room_name = room_name
+        self.client_infos = []
+
```

Comment:
チャットルームの情報が分かるように、__repr__を定義してもいいかもしれないですね。
```suggestion
    def __repr__(self):
        repr_string = ""
        for client_info in self.client_infos:
            repr_string += f"<ClientInfo(udp_addr={client_info.udp_addr}, tcp_addr={client_info.tcp_addr}, access_token={client_info.access_token}, username={client_info.username}, is_host={client_info.is_host})>"

        return repr_string
```

Reply:
実装しました！ClientInfoクラスで__repr__が定義されているので、以下で動きました！
```
def __repr__(self):
        return (
            f"<ChatRoom(room_name={self.room_name}, client_infos={self.client_infos})>"
        )
```